model/proxy/alphav1/config/cfg.proto

```diff
@@ -21,230 +21,162 @@ package istio.proxy.v1alpha.config;
 option go_package = "config";
 
 // Glossary & concepts
-
-// Cluster - set of pods/VMs/containers where the service is running. The
-// members of a cluster share one or more common attributes. For e.g., all
-// pods in a cluster could share a common set of labels, or be running the
-// same version of the application binary.
-
-// Multiple clusters/service - In a continuous deployment scenario, for a
-// given service, there can be multiple clusters running potentially
-// different variants of the application binary. These variants are not
-// necessarily different API versions. They could be iterative changes to
-// the same service, deployed in different environments (prod, staging,
-// dev, etc.). Common scenarios where this occurs include A/B testing,
-// canary rollouts, etc. The choice of a particular cluster can be decided
-// based on various criterion (headers, url, etc.) and/or by weights
-// assigned to each cluster.
-
-// Cluster identifier: A name and one or more tags uniquely identify each
-// cluster in a service. Names and tags can be arbitrary strings. Parsing
-// of names and tags is specific to the platform (e.g., in kubernetes, name
-// could correspond to the service Name, while each tag string could be
-// parsed into a pod label (key:value))
-
-// Downstream service/cluster - client (browser or another service cluster)
-// calling the proxy/sidecar (typically to reach another service).
-
-// Upstream service - The remote service to which the proxy/sidecar is
-// talking to, on behalf of the Downstream service. There can be one or
-// more upstream clusters for a given upstream service (see the discussion
-// on versions above). The proxy would choose the upstream cluster based on
-// various routing rules.
-
-// Applications address only the upstream service using without knowledge
-// of individual upstream clusters. The actual choice of the cluster is
-// determined by the proxy, enabling the application code to decouple
-// itself from the evolution of dependent services.
+//
+// Service instances - set of pods/VMs/containers where the service is running.
+// The members share one or more common attributes. For e.g., all pods in a
+// group could share a common set of labels, or be running the same version of
+// the application binary.
```

Comment:
This seems confusing. I would say service instances share only one thing - they are all implementing some variation of the same service. I also think that before defining the concept of Service instances, we need to clearly define the term "Service" itself. Specifically, explain that a service is a name (string) of some functionality (e.g., ServiceA) representing part of an application, and point out that clients use this name to refer to the functionality being called.